### PR TITLE
English: About Us

### PR DESCRIPTION
**Pull request by RonaldDijkstra**
Translates about us

### CODE REVIEW

File: source/localizable/over-ons/_team.erb [hunk 1]
```diff
@@ -16,13 +16,14 @@
...
     <li class="you">
+
       <% locale_link_to("/jobs/index.html") do %>
         <div class="avatar"><%= t("about.team.you") %></div>
         <h5>
           <span><%= t("about.team.work_with_us") %></span>
           <span>Defacto</span>
         </h5>
-      <% end %>
+      <% end unless root_locale == :en %>
```
Comment (fatboypunk) [inline source/localizable/over-ons/_team.erb]: Think it's better to do a if block around this

File: source/localizable/over-ons/_over-ons.en.md.erb [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+Choosing quality over the maximisation of profit, choosing an interesting job over the monotone, investing in people and guaranteeing continuity and growth, that is what we stand for.
+
+<%= image_tag "defacto-pand.jpg", title: "Defacto", alt: "Defacto", class: "center" %>
+
+Defacto has a youthful, open and informal business culture where there is a lot of room for creativity and responsibility. Our motto is 'Developing People'. Investing in our developers, consultants and sales &amp; support staff is central for us. From this basis we develop software that enables our clients to get the best from our services.
+
+Defacto is an quintessential example of a software company. New technology, technical challeges and delivering quality is what excites us. Offering our clients tailored solutions and sparing them a lot of time is our mission.
+
+We are an organisation with flat hierarchy and a high level of transparency.Defacto is also active in the German market since 2014.
```
Comment (fatboypunk) [inline source/localizable/over-ons/_over-ons.en.md.erb]: missing a space 
`transparency.Defacto `

File: source/localizable/over-ons/_team.erb (hunk 1 above)
Comment (RonaldDijkstra) [inline source/localizable/over-ons/_team.erb]: ok!

File: source/localizable/over-ons/_over-ons.en.md.erb (hunk 2 above)
Comment (RonaldDijkstra) [inline source/localizable/over-ons/_over-ons.en.md.erb]: thx!

**Pull request merged**